miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/permission/MiniAppPermission.kt

```diff
@@ -3,6 +3,7 @@ package com.rakuten.tech.mobile.miniapp.permission
 import androidx.annotation.Keep
 
 /** Type of miniapp permission. **/
+@Deprecated("Use MiniAppDevicePermissionType instead")
 enum class MiniAppPermissionType(val type: String) {
```

Comment:
Couldn't you use a `typealias` here? And then you don't need to have two duplicate implementations:
```
typealias MiniAppPermissionType = MiniAppDevicePermissionType
```

Reply:
Yeah, I'll update this part.